### ISSUE TITLE
Debugging notice displayed on game page

### ISSUE THREAD

**Issue by sbourget** [xow/moodle-mod_quizgame#35]
The view page displays the following debugging notice:

Too many params passed to js_call_amd("mod_quizgame/quizgame", "init")

    line 1025 of /lib/outputrequirementslib.php: call to debugging()
    line 61 of /mod/quizgame/renderer.php: call to page_requirements_manager->js_call_amd()
    line 70 of /mod/quizgame/view.php: call to mod_quizgame_renderer->render_game()

This is being displayed on the current master (3.5) build of Moodle. (I have't tried it on Moodle 3.4 yet)

**Comment by xow** [comment]
Yeah this is if you have too many questions in the selected question bank (which is not that many). I'm not sure of a good to pass the data without passing it through js_call_amd.

Perhaps just rendering some js directly... cooking? not sure.

I suppose the ideal is an ajax endpoint that we load the questions from.
